### DISCUSSION TITLE
Suggestion for renaming this add-on to support more sources in the feature

### DISCUSSION THREAD

**Discussion by mhdzumair** [mhdzumair/tamilblasters_stremio_addon#37] [General]
Hi everyone. As of many request to add more sites to scraping source. I decided to rename this add-on or create a new add-on in favour of scraping **torrent** movies & series. So provide your suggestions

**Comment by KARANRSB** [comment]
Add Tamilmv also

**Comment by mhdzumair** [comment]
Planning to add TamilMV but suggest me a new name for add-on

**Comment by KARANRSB** [comment]
A new name can be SouthTorrentio it's the best I could come up with.

**Reply by jrjaggari** [reply]
IndianScraper or just TorrentScraper

**Comment by mhdzumair** [comment]
How about CineFusion

**Reply by jrjaggari** [reply]
How about MediaFusion so it covers series and movies?

**Reply by mhdzumair** [reply]
Sounds interesting

**Reply by mhdzumair** [reply]
Anyone want to contribute for creating a logo for the new Add-on `MediaFusion` ?

**Reply by mhdzumair** [reply]
Logo can be created manually or using AI tools. I'm okay with whatever simple design.

**Comment by KARANRSB** [comment]
Yeah it's good but it sound more Indian nah

**Reply by mhdzumair** [reply]
Im planning to add many torrents sources. So its not only the indian movies.

**Comment by KARANRSB** [comment]
Try a more Indian name guys.
All the names are good which u have given but it should sound more like our
country based as there are more global addons on stremio so.

**Reply by jrjaggari** [reply]
I think the intent is to be more global. I would replace torrentio with this plugin if this has everything torrentio has + Indian!

**Comment by KARANRSB** [comment]
Oh wow u are trying to make it like torrentio and better?
Wow nice bro then I'll think and let u know a good name?

**Comment by mhdzumair** [comment]
New project: https://github.com/mhdzumair/MediaFusion

**Comment by jjose3** [comment]
Great work on enabling real-debrid support. Can we get some info on if the links are already cached on real-debrid? Torrentio addon has this feature which shows as "RD+" for cached links and "RD Download" for uncached links

**Comment by naveenland4** [comment]
https://i.imgur.com/pzJ2mqO.png

Can we use this as logo? A simple one...

**Comment by KARANRSB** [comment]
The tamilblasters addon isn't working for me?

**Reply by mhdzumair** [reply]
Yes. They added Cloudflare validation.

**Comment by KARANRSB** [comment]
So what to do now?And who added cloudflare by the way also I get this
whenever I try to view a movie which I know has already been downloaded for
debrid or which has already been seeded.

**Reply by mhdzumair** [reply]
Im not affiliated with TamilBlasters or any of the site. It's site admins who are adding that to restrict for web scrapping. I just only web scraping the movies and series. 

Now its working again.
